src/components/DownloadComponents/UpcomingReleases/UpcomingReleasesItem/index.tsx

```diff
@@ -13,6 +14,11 @@ const UpcomingReleasesItem = ({
   alreadyReleased,
 }: Props): JSX.Element => {
   const Image = alreadyReleased ? HexagonFilled : HexagonOutline;
+  const Released = alreadyReleased ? (
```

Comment:
That is not an optimal ICU translation.

I would recommend:

- The translation key to be "components.upcomingReleasesItem.releaseStatus"
- The translation bag to be `{release, select, true {Released}other {}} {releaseDate}`

This would simplify the code:

```jsx
<p className={styles.releaseDate}>
  <FormattedMessage id="components.UpcomingReleasesItem" values={{ released: alreadyReleased, releaseDate }} />
</p>
```

Reply:
Ok i will do that thanks for reviewing

Reply:
Your simplify code isn't working...

Reply:
My code is an example. Even tho, I don't think my code is the problem. If you commit the changes I could review what you did wrong.

Reply:
(You should be able to read the docs of react-intl and check why it is not working).